sdk/keyvault/Azure.Security.KeyVault.Keys/README.md

```diff
@@ -81,13 +81,26 @@ Key key = await Client.CreateKey("key-name", KeyType.EllipticCurve);
 > new DefaultAzureCredential():
 > Uses the environment variables previously set (`AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, and `AZURE_TENANT_ID`).
 
+#### Create CryptographyClient
+Once you've created a `Key` in the key vault, you can also create the [CryptographyClient][crypto_client_class]:
+
```

Comment:
Could add reference to how to create a key here, "Learn how to [create a key] [create-a-key-sample-reference-link-here]  in the key vault."

Reply:
I didn't add creating the key in this snippet because it is in the snippet directly above this.